src/zeta/aca_oam_server.cpp

```diff
@@ -262,20 +252,17 @@ int ACA_Oam_Server::_add_direct_path(oam_match match, oam_action action)
 {
   unsigned long not_care_culminative_time;
   int overall_rc = EXIT_SUCCESS;
-  //
 
   string vlan_id = to_string(aca_vlan_manager::ACA_Vlan_Manager::get_instance().get_or_create_vlan_id(
-          std::stoul(match.vni)));
-  string outport_name =
-          aca_get_outport_name(alcor::schema::NetworkType::VXLAN, action.node_nw_dst);
+          match.vni));
 
   string cmd_match = "ip,nw_proto=" + match.proto + ",nw_src=" + match.sip +
                      ",nw_dst=" + match.dip + ",tp_src=" + match.sport +
                      ",tp_dst=" + match.dport + ",dl_vlan=" + vlan_id;
-  string cmd_action = ",actions=\"strip_vlan,load:" + match.vni +
-                      "->NXM_NX_TUN_ID[],mod_dl_dst=" + action.inst_dl_dst +
-                      ",mod_nw_dst=" + action.inst_nw_dst +
-                      ",output:" + outport_name + "\"";
+  string cmd_action = ",actions=\"strip_vlan,load:" + to_string(match.vni) +
+                      "->NXM_NX_TUN_ID[],set_field:" + action.node_nw_dst +
+                      "->tun_dst,mod_dl_dst=" + action.inst_dl_dst +
+                      ",mod_nw_dst=" + action.inst_nw_dst + ",output:vxlan-generic\"";
```

Comment:
do you need the below? I believe the out going packet already has the correct information. 

mod_dl_dst=" + action.inst_dl_dst + ",mod_nw_dst=" + action.inst_nw_dst

Reply:
don't worry about it as discussed.